sql/core/pom.xml

```diff
@@ -130,6 +130,12 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.apache.avro</groupId>
```

Comment:
Ah sorry I missed this, yes we should get this in ASAP.

Is this the only module that would need this updated dependency? we could declare this in the parent's `dependencyManagement` section in `test` scope.

Does the "1.8.0" need to sync with a Parquet or other Avro versions -- is there or should there be a property tying them together?